src/agent/sqlite_manager/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.22)
+
+set(CMAKE_TOOLCHAIN_FILE "${CMAKE_SOURCE_DIR}/../../vcpkg/scripts/buildsystems/vcpkg.cmake")
+set(VCPKG_MANIFEST_DIR ${CMAKE_SOURCE_DIR}/../../)
+
+project(SQLiteManager)
+
+include(../../cmake/CommonSettings.cmake)
+set_common_settings()
+
+find_package(fmt REQUIRED)
+find_package(SQLiteCpp REQUIRED)
+
+add_library(SQLiteManager src/sqlite_manager.cpp)
+
+target_include_directories(SQLiteManager PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include PRIVATE ${SQLiteCpp_INCLUDE_DIRS})
+target_link_libraries(SQLiteManager PUBLIC SQLiteCpp fmt::fmt)
+
```

Comment:
Missing these lines here (check the other cmake files):
```
include(../../cmake/ConfigureTarget.cmake)
configure_target(AgentInfo)
```

Reply:
Done